fastlane/Fastfile

```diff
@@ -36,6 +36,29 @@ def get_required_env(key)
   ENV[key]
 end
 
+# Helper function to DRY the code to build the app
+def build_simplenote(codesign:, archive_path: nil)
```

Comment:
I've never done or seen this on a `Fastfile`, but I liked the idea of using the keyword arguments to make it clear to the caller what values to pass. As opposed to using a private lane, where it's not clear (to me) what to pass every time.

What do you think? Too original?

Reply:
Yeah, tbh I never really understood why fastlane invented the concept of private_lane when using regular functions via `def` already provides the same functionality 🤷 

The only difference of using `def` vs using `private_lane` at runtime is that you will see "Cruising to private lane XYZ" in the log/output when you were running your main lane and it calls out to the private one; then "Cruising back to lane ABC" once it's back. While a call to a function won't be logged into the fastlane output. So, each solution has its pros and cons 🤷‍♂️